api/sessionrecording/session_recording.go

```diff
@@ -0,0 +1,303 @@
+/*
+Copyright 2024 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package sessionrecording
```

Comment:
Let's add the package-level godoc here.

```suggestion
// Package sessionrecording can be used to read Teleport's
// session recording files.
package sessionrecording
```

Reply:
Thank you, added